### PR TITLE
Sync docs and anagram tests

### PR DESCRIPTION
**Pull request by IsaacG**

### CODE REVIEW

File: exercises/practice/anagram/test-anagram.bats [hunk 1]
```diff
@@ -147,9 +147,7 @@ WORD_LIST
     [[ $BATS_RUN_SKIPPED == "true" ]] || skip
 
     run gawk -f anagram.awk -v key=go <<WORD_LIST
-go
-Go
-GO
+go Go GO
```
Comment (glennj) [inline exercises/practice/anagram/test-anagram.bats:150]:
```suggestion
goGoGO
```

This seems to be what the canonical data is giving:
```json
    {
      "uuid": "630abb71-a94e-4715-8395-179ec1df9f91",
      "reimplements": "7cc195ad-e3c7-44ee-9fd2-d3c344806a2c",
      "description": "does not detect an anagram if the original word is repeated",
      "property": "findAnagrams",
      "input": {
        "subject": "go",
        "candidates": ["goGoGO"]
      },
      "expected": []
    },
```

Comment (IsaacG) [reply exercises/practice/anagram/test-anagram.bats:150]: Fixed in the latest commit.

Comment (glennj) [reply exercises/practice/anagram/test-anagram.bats:150]: Did you push it?

Comment (IsaacG) [reply exercises/practice/anagram/test-anagram.bats]: I did! But to the wrong branch :facepalm: 

How about now?

**Review by glennj — APPROVED** [review]

**Pull request merged**